src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/index.tsx

```diff
@@ -293,6 +17,12 @@ const SamplesTable = ({
   isLoading,
   setCheckedSamples,
 }: Props): JSX.Element => {
+  const pathogen = useSelector(selectCurrentPathogen);
+  const columns: ColumnDef<Sample, any>[] = useMemo(
```

Comment:
👍 , I wonder if I should have been memoizing other config getters

Reply:
if you define the columns outside the component, you don't have to. I only have done it here since I need to put it inside the component in order to pass the pathogen in